src/pages/storage/StorageVolumeDetail.tsx

```diff
@@ -48,7 +50,9 @@ const StorageVolumeDetail: FC = () => {
     isLoading,
   } = useQuery({
     queryKey: [queryKeys.storage, pool, project, type, volumeName],
-    queryFn: () => fetchStorageVolume(pool, project, type, volumeName),
+    queryFn: () =>
+      loadCustomVolumeAndSnapshots({ pool, project, type, volumeName }),
```

Comment:
Why load the snapshots here as well? I think we only need the volume here and can load snapshots inside the `StorageVolumeSnapshots` component. I think they are only needed there.

Reply:
This is my bad, I referenced the `InstanceDetail` logic and didn't think it through. The request to get snapshots is now moved to the `StorageVolumeSnapshots` component.